### PR TITLE
fix UI tests after changes on the UI side

### PR DESCRIPTION
**Pull request by AndreeaDora**

### CODE REVIEW

File: scripts/protractor/spec.js [hunk 1]
```diff
@@ -237,17 +237,15 @@ describe('ILM', function() {
         element(sy.createImageLocationPublicReg).click();
         browser.wait(protractor.ExpectedConditions.visibilityOf(element(sy.createImageNameSearch), 60000));
         element(sy.createImageNameSearch).click();
-        browser.wait(protractor.ExpectedConditions.visibilityOf(element(sy.createImageNameSearch), 60000));
         element(sy.createImageNameSearch).sendKeys(config.imageName);
-        browser.wait(protractor.until.elementLocated(by.className('description')), 180000);
+        /*browser.wait(protractor.until.elementLocated(by.className('description')), 180000);
```
Comment (soninob) [inline scripts/protractor/spec.js]: Do we need the comment code?
If not, let's remove it.

Comment (AndreeaDora) [inline scripts/protractor/spec.js]: @soninob After the guys from docker implement the search for v2 we can use it.

Comment (soninob) [inline scripts/protractor/spec.js]: So please add a comment explaining that, thanks.

**Comment by AndreeaDora** [comment]
There is another pull request opened with the requested comment.

**Pull request closed without merging**